consensus/wal.go

```
@@ -95,7 +95,9 @@ func (wal *WAL) Save(wmsg WALMessage) {
 }
 
 func (wal *WAL) writeEndHeight(height int) {
-	wal.group.WriteLine(Fmt("#ENDHEIGHT: %v", height))
+	if err := wal.group.WriteLine(Fmt("#ENDHEIGHT: %v", height)); err != nil {
+		panic(err)
```

Not sure we want to panic here either. Maybe this should return the error